_sass/_nested-nav.scss

```diff
@@ -0,0 +1,130 @@
+#top {
+  z-index: 9;
+}
+
+@media screen and (min-width: 46.25rem) {
```

Comment:
Leaving these magic numbers in so this file remains identical to the one on the website for ease of update.